stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/controller/QueryChannelsController.kt

```diff
@@ -50,5 +50,5 @@ public interface QueryChannelsController {
      */
     public val loadingMore: LiveData<Boolean>
 
-    public fun addChannelIfFilterMatches(channel: Channel)
+    public suspend fun addChannelIfFilterMatches(channel: Channel)
```

Comment:
Same as above, this is public suspending API now, not sure if this is a great idea.

We might be able to make this one internal though. Maybe it could be a method that's only on the `Impl` class but not on the interface?

Reply:
Again same thing. Useless interface. I'd like to remove it and use it only internally

Reply:
Sounds good, let's do that then 👍